File: lang/python/src/fun/vac/algorithms/meta/Sorting.py
from fun.vac.algorithms.sorting import BubbleSort
from fun.vac.algorithms.sorting import InsertionSort
from fun.vac.algorithms.sorting import MergeSort
from fun.vac.algorithms.sorting import QuickSort
from fun.vac.algorithms.sorting import SelectionSort

def bubbleSort(arr: list) -> None:
    BubbleSort.sort(arr)

def insertionSort(arr: list) -> None:
    InsertionSort.sort(arr)

def mergeSort(arr: list) -> None:
    MergeSort.sort(arr)

def quickSort(arr: list) -> None:
    QuickSort.sort(arr)

def selectionSort(arr: list) -> None:
    SelectionSort.sort(arr)

File: lang/java/src/fun/vac/algorithms/meta/Sorting.java
package fun.vac.algorithms.meta;

import fun.vac.algorithms.sorting.BubbleSort;
import fun.vac.algorithms.sorting.InsertionSort;
import fun.vac.algorithms.sorting.MergeSort;
import fun.vac.algorithms.sorting.QuickSort;
import fun.vac.algorithms.sorting.SelectionSort;

public class Sorting {

    private Sorting() {}

    public static <T extends Comparable<T>> void bubbleSort(T[] arr) {
        BubbleSort.sort(arr);
    }

    public static <T extends Comparable<T>> void insertionSort(T[] arr) {
        InsertionSort.sort(arr);
    }

    public static <T extends Comparable<T>> void mergeSort(T[] arr) {
        MergeSort.sort(arr);
    }

    public static <T extends Comparable<T>> void quickSort(T[] arr) {
        QuickSort.sort(arr);
    }

    public static <T extends Comparable<T>> void selectionSort(T[] arr) {
        SelectionSort.sort(arr);
    }
}
